topics/genome-annotation/tutorials/helixer/tutorial.md

```diff
@@ -222,6 +222,35 @@ This gives information about the completeness of the Helixer annotation. A good
 >
 {: .comment}
 
+## Evaluation with **OMArk**
+
+[OMArk](https://github.com/DessimozLab/OMArk) is proteome quality assessment software. It provides measures of proteome completeness, characterises the consistency of all protein-coding genes with their homologues and identifies the presence of contamination by other species. OMArk is based on the OMA orthology database, from which it exploits orthology relationships, and on the OMAmer software for rapid placement of all proteins in gene families.
+
+OMArk's analysis is based on HOGs (Hierarchical Orthologous Groups), which play a central role in its assessment of the completeness and coherence of gene sets. HOGs make it possible to compare the genes of a given species with groups of orthologous genes conserved across a taxonomic clade. 
+
+> <hands-on-title>OMArk on extracted protein sequences</hands-on-title>
+>
+> 1. {% tool [OMArk](toolshed.g2.bx.psu.edu/repos/iuc/omark/omark/0.3.0+galaxy2) %} with the following parameters:
```

Comment:
should this be added to the workflow as well?

Reply:
Yes, it should be added to the workflow.